Eat_and_Greet/src/screens/OnboardingScreens/emailVerify.tsx

```diff
@@ -20,7 +48,8 @@ function EmailVerify() {
                         Didn’t get a code?
```

Comment:
If this is here always, get rid of the fail case of send email. Just have it send the email if it is requested, if they click the button twice it makes sense to send another email (you could try to time gate it but that seems hard).